exoctk/exoctk_app/app_exoctk.py

```diff
@@ -490,9 +486,9 @@ def contam_visibility():
             # Make plot
             TOOLS = 'crosshair, reset, hover, save'
             fig = figure(tools=TOOLS, plot_width=800, plot_height=400, x_axis_type='datetime', title=title)
-            fh = StringIO()
-            table.write(fh, format='ascii')
-            visib_table = fh.getvalue()
+            #fh = StringIO()
```

Comment:
@jaymedina should we remove the ascii table code since we are writing out a csv now?